scripts/shared_colors.shader

```diff
@@ -81,7 +81,7 @@ textures/shared_colors/red
 	surfaceparm nomarks
 
 	{
-		map textures/shared_colors_src/red_d
+		map $red
```

Comment:
If you ask me `_red` should be nuked. Just use `rgbgen const (1 0 0)`

Reply:
Now all the `colors` shaders use either:

- `$white` (white)
- `$black` (black)
- `$white` + `rgbgen const` (all remaining colors)
- no stage (fully transparent)

None of the `colors` images are now expected to be loaded by the Dæmon engine. They are shipped to be used as `qer_editorImage` which is meant to be used by both the NetRadiant editor end the Q3map2 map compiler (`qer_editorImage` being the only syntax meant to be understood by them).